packages/ui/src/components/ListV2/index.tsx

```diff
@@ -47,50 +30,59 @@ type ListProps<T> = {
    * Auto close opened expandable row when another is opening
    * */
   autoClose?: boolean
-
+  showExpandArrow?: boolean
   columns?: ListColumn[]
   template?: string
   className?: string
 }
+type ListProps = XOR<
+  [
+    CommonListProps,
+    CommonListProps & {
+      // Multi Select props
+      selectedIds: string[]
+      onSelectedIdsChange: (selectedIds: string[]) => void
+    },
+  ]
+>
 
-export const List = <T = ListDataObject,>({
+export const List = ({
   children,
   columns,
   template,
-  isSelectable,
   selectedIds,
   onSelectedIdsChange,
-  data,
-  idKey,
   isLoading,
   autoClose,
+  showExpandArrow = false,
```

Comment:
Shouldn't `showExpandArrow` be defined by the list itself when there is an Expandable children?

Reply:
Acutally we doesn't know if all our expandable rows will have an arrow, may be @fabienhebert talk about that with @PetmePant  ? 🤔